app/models/invite_email_mentor.rb

```diff
@@ -8,7 +8,9 @@ def mail_to_send
   end
 
   def send!(force_send: false)
-    can_send_automatically = user.is_on_core_induction_programme? && Time.zone.now >= INVITES_SENT_FROM
+    return unless user.is_on_core_induction_programme?
+
+    can_send_automatically = Time.zone.now >= INVITES_SENT_FROM
```

Comment:
Was the force send sending non-cip users an email before making this change?

Reply:
It in principle could, yes.

Reply:
And I got worried that I would force-send emails to a wrong user by accident :P